.github/workflows/dependency-review.yaml

```diff
@@ -36,5 +36,5 @@ jobs:
         uses: actions/dependency-review-action@v3
         with:
           fail-on-severity: moderate
-          allow-licenses: 0BSD, Apache-2.0, BlueOak-1.0.0, BSD-2-Clause, BSD-2-Clause-FreeBSD, BSD-3-Clause, CC-BY-3.0, CC-BY-4.0, CC0-1.0, ISC, LGPL-2.1, MIT, MIT-0, MPL-2.0, ODC-By-1.0, OFL-1.1, Python-2.0, Unicode-DFS-2016, Unlicense, Zlib, (MIT OR Apache-2.0) AND Unicode-DFS-2016, Apache-2.0 AND BSD-3-Clause, ISC AND MIT, MIT AND Zlib, MIT AND BSD-3-Clause
+          allow-licenses: 0BSD, Apache-2.0, BlueOak-1.0.0, BSD-2-Clause, BSD-2-Clause-FreeBSD, BSD-3-Clause, CC-BY-3.0, CC-BY-4.0, CC0-1.0, ISC, LGPL-2.1, MIT, MIT-0, MPL-2.0, ODC-By-1.0, OFL-1.1, Python-2.0, Unicode-DFS-2016, Unlicense, Zlib, (MIT OR Apache-2.0) AND Unicode-DFS-2016, Apache-2.0 AND BSD-3-Clause, ISC AND MIT, MIT AND Zlib, MIT AND BSD-3-Clause AND IICL-1.1
```

Comment:
```suggestion
          allow-licenses: 0BSD, Apache-2.0, BlueOak-1.0.0, BSD-2-Clause, BSD-2-Clause-FreeBSD, BSD-3-Clause, CC-BY-3.0, CC-BY-4.0, CC0-1.0, ISC, LGPL-2.1, MIT, MIT-0, MPL-2.0, ODC-By-1.0, OFL-1.1, Python-2.0, Unicode-DFS-2016, Unlicense, Zlib, (MIT OR Apache-2.0) AND Unicode-DFS-2016, Apache-2.0 AND BSD-3-Clause, ISC AND MIT, MIT AND Zlib, MIT AND BSD-3-Claus, IICL-1.1
```

Reply:
I added this in another commit, I'm still getting

```
Run actions/dependency-review-action@v3
  with:
    fail-on-severity: moderate
    allow-licenses: 0BSD, Apache-[2](https://github.com/gravitational/shared-workflows/actions/runs/6015942188/job/16318870939#step:3:2).0, BlueOak-1.0.0, BSD-2-Clause, BSD-2-Clause-FreeBSD, BSD-[3](https://github.com/gravitational/shared-workflows/actions/runs/6015942188/job/16318870939#step:3:3)-Clause, CC-BY-3.0, CC-BY-[4](https://github.com/gravitational/shared-workflows/actions/runs/6015942188/job/16318870939#step:3:4).0, CC0-1.0, ISC, LGPL-2.1, MIT, MIT-0, MPL-2.0, ODC-By-1.0, OFL-1.1, Python-2.0, Unicode-DFS-2016, Unlicense, Zlib, (MIT OR Apache-2.0) AND Unicode-DFS-2016, Apache-2.0 AND BSD-3-Clause, ISC AND MIT, MIT AND Zlib, MIT AND BSD-3-Clause, IICL
    repo-token: ***
Error: Invalid license(s) in allow-licenses: IICL
```

Reply:
I believe its checking against this list: https://spdx.org/licenses/

Reply:
How do we whitelist / skip the check for a non SPDX license?

Reply:
Tracing some work from @jentfoo:

https://github.com/actions/dependency-review-action/issues/385

Leads me to:

https://github.com/actions/dependency-review-action/pull/423

`allow-dependencies-licenses` should get us there.

I'd recommend we skip trying to add the proprietary license, and instead say this dependency doesn't need a licence check. Try adding the following config:

```
allow-dependencies-licenses:
  - pkg:npm/%40inkeep/widgets
```

Reply:
Thanks Walt, I think I fixed it.